### PR TITLE
Fixing Broken TokenAllowance Story

### LINKED ISSUE #17343: Broken Story : TokenAllowance
## Description
The test-story integration has exposed some broken stories. They were broken before but now they are showing up in the test-storybook pipeline which is good! In the meantime, I've updated all broken stories to be prefixed with .stories-to-do.js so they don't show up in the storybook. Let's fix these currently broken stories. 

### Technical Details
Rename `ui/pages/token-allowance/token-allowance.stories-to-do.js `to `ui/pages/token-allowance/token-allowance.stories.js`
Fix `ui/pages/token-allowance/token-allowance.stories.js`

### Acceptance Criteria
TokenAllowance story works

### PR DESCRIPTION
**Pull request by Jainex17**
## Explanation

The `TokenAllowance` story was not being displayed in the Storybook. This PR resolves the issue and now it is shown in the Storybook.

### chnages made
- Renamed `token-allowance.stories-to-do.js` to `token-allowance.stories.js`.
- Fixed issues in `token-allowance.stories.js`.
- Added `AllBannerAlert` story to display all alerts

* Fixes #17343 

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/814cf523-790e-4f36-9cea-33c95dee5d43

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/bd5e8dc0-b759-4f42-9e57-166a076ddcfd

## Manual Testing Steps

- Pull this branch
- Run storybook
- Search `TokenAllowance`

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

### CODE REVIEW

**Comment by Jainex17** [comment]
@NidhiKJha Could you please review this pr?

**Review by NidhiKJha — DISMISSED** [review]
LGTM! Thanks for your contribution

**Comment by Jainex17** [comment]
@georgewrmarshall Could you please review this pr?

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Thanks for your contribution @Jainex17

**Comment by georgewrmarshall** [comment]
Still a few more selector updates that need to be made to get this to pass. Let me know if you need some assistance @Jainex17

**Comment by Jainex17** [comment]
> Still a few more selector updates that need to be made to get this to pass. Let me know if you need some assistance @Jainex17


Thanks, I'm still making an effort, but having some steps from you to resolve this build issue would be incredibly helpful.

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Hey @Jainex17, to get this PR in I reverted the changes to the token allowance files and we can open another PR to swap out the deprecated components. Thanks for your contribution

**Review by NidhiKJha — DISMISSED** [review]
LGTM

File: ui/pages/token-allowance/token-allowance.stories.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    ...testData.metamask,
+    // TODO: Mock state correctly to show insufficient funds error
+    //
+    accounts: {
+      ...testData.metamask.accounts,
+      '0x9d0ba4ddac06032527b140912ec808ab9451b788': {
+        balance: '0x0',
+      },
+    },
+  },
```
Comment (georgewrmarshall) [inline ui/pages/token-allowance/token-allowance.stories.js:235]: Hey @matthewwalsh0, do you have any ideas on how one could show the insufficient funds for gas error? I seem to be struggling with that

Screenshot 2023-10-16 at 3 07 43 PM

**Comment by kumavis** [comment]
thank you for this!
the After video shows a bug suggesting some type error: "Spending cap request for your [Object object]"

**Review by kumavis — CHANGES_REQUESTED** [review]
thank you for this!
the After video shows a bug suggesting some type error: "Spending cap request for your [Object object]"

**Comment by georgewrmarshall** [comment]
> I created a follow-up ticket for us to address here: #21405
Thanks @digiwand  🙏

**Review by kumavis — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Review by legobeat — APPROVED** [review]

**Comment by Jainex17** [comment]
@georgewrmarshall Sorry for being inactive is there anything i can do in this PR Thanks

**Pull request merged**